web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -1011,11 +1006,11 @@ GROUP BY 1`;
               ),
             },
             {
-              Header: twoLines('Segment size (MB)', 'min / avg / max'),
+              Header: twoLines('Segment size (rows)', 'minimum / average / maximum'),
               show: hiddenColumns.exists('Segment size'),
               accessor: 'avg_segment_size',
               filterable: false,
-              width: 150,
+              width: 220,
```

Comment:
Is this wide enough to handle 7 digits for min, avg and max?

From the doc linked in this PR `it's generally recommended for each segment to have around 5 million rows`
Do you think it's users would want to see a rounded number instead - something like `2.1M` instead of `2,105,248`
^ just a thought

Reply:
I don't have a strong opinion on this, and will defer to your judgement.

The risk with rounding to millions is that for smaller datasources, it could always show `0.000 M`

I wonder if the UI could use the largest rounding per datasource. So you could have something like

```
DS1    5.343 M, 0.004 M, 0.000 M
DS2    0.320 k, 0.120 k, 321.1 k
``` 

^ even in this example, I'm really confused about how to pick whether to keep it as `321.1 k` or `0.321 M`

Number formatting isn't my forte 🙃

Reply:
What do you think about

![image](https://user-images.githubusercontent.com/177816/95694139-37695b80-0be5-11eb-9f47-c9ae8a277732.png)

The only numbers that would be rendered as `0.000 M` would be numbers `<= 500` so we could display them directly.
Side note: There is a style pass happening on a different branch that would change the font to one that has monospaced numbers

Reply:
👍